### PR TITLE
replace / with - in job progress tracker

### PR DESCRIPTION
**Pull request by wdbaruni**
For job level events, we now print `----------` instead `/` under executionDI

```
 TIME          EXEC. ID    TOPIC            EVENT
 09:12:32.194  /           Submission       Job submitted
 09:12:32.216  e-a201aa76  Scheduling       Requested execution on n-f057ce5e
 09:12:32.257  e-a201aa76  Execution        Running
 09:12:32.475  e-a201aa76  Execution        Completed successfully
```

```
 TIME          EXEC. ID    TOPIC            EVENT
 14:08:03.138  ----------  Submission       Job submitted
 14:08:03.149  e-d3860563  Scheduling       Requested execution on n-c5d27044
 14:08:03.202  e-d3860563  Execution        Running
 14:08:06.614  e-d3860563  Execution        Completed successfully
```

Closes https://github.com/bacalhau-project/bacalhau/issues/4505

### CODE REVIEW

**Comment by wdbaruni** [comment]
@aronchick to clarify, we have some events for job level state update and others for execution level. Mostly execution updates. I've replaced `/` with `----------` as suggested in #4505 , but let me know if you have other suggestions now the content is clearer

**Comment by wdbaruni** [comment]
To be honest it doesn't look easier on the eye when querying all history events for a job, such as :

```
Walid-MacBook-5: ~
→       bacalhau job history j-c7bf0dac-5981-4147-9d16-301b5e132613 --no-style
 TIME                 LEVEL           EXEC. ID    TOPIC         EVENT
 Sep 25 14:08:03.138  JobLevel        ----------  Submission    Job submitted
 Sep 25 14:08:03.149  ExecutionLevel  e-d3860563  Scheduling    Requested execution on n-c5d27044
 Sep 25 14:08:03.202  JobLevel        ----------  State Update  Running
 Sep 25 14:08:03.202  ExecutionLevel  e-d3860563  Execution     Running
 Sep 25 14:08:06.614  ExecutionLevel  e-d3860563  Execution     Completed successfully
 Sep 25 14:08:06.627  JobLevel        ----------  State Update  Completed
```

**Comment by aronchick** [comment]
yeah - maybe just blank? The problem is '\' looked like something broken

**Review by aronchick — APPROVED** [review]
maybe just blank?

File: cmd/util/cols/history.go [hunk 1]
```diff
@@ -34,7 +35,7 @@ var (
 		ColumnConfig: table.ColumnConfig{Name: "Exec. ID", WidthMax: 10, WidthMaxEnforcer: text.WrapText},
 		Value: func(j *models.JobHistory) string {
 			if j.ExecutionID == "" {
-				return "/"
+				return strings.Repeat("-", 10)
```
Comment (frrist) [inline cmd/util/cols/history.go:38]: might want to extract `10` to a constant given it must match the column max width

**Review by frrist — APPROVED** [review]

**Pull request merged**